listenbrainz/mbid_mapping/mapping/search.py

```diff
@@ -0,0 +1,39 @@
+
+import typesense
+import typesense.exceptions
+from mapping.typesense_index import prepare_string
+
+import config
+
+
+def search(query):
+
+    client = typesense.Client({
+        'nodes': [{
+          'host': config.TYPESENSE_HOST,
+          'port': config.TYPESENSE_PORT,
+          'protocol': 'http',
+        }],
+        'api_key': config.TYPESENSE_API_KEY,
+        'connection_timeout_seconds': 2
+    })
+
+    search_parameters = {
+        'q': prepare_string(query),
+        'query_by': "combined",
```

Comment:
eventually it might be nice to move these parameters to config.py if we still want to be able to tweak them

Reply:
:+1: